cmd/minishift/cmd/root.go

```diff
@@ -73,6 +75,8 @@ var RootCmd = &cobra.Command{
 			isAddonInstallRequired bool
 		)
 
+		checkProfileExist(cmd)
```

Comment:
I think validateProfileExists() would be a better name.

Reply:
@LalatenduMohanty For me `checkProfileExist` make more sense because we are just checking if profile exist or not but when we say `validateProfileExists` then what we validating in profile?